File: 1791-richest-customer-wealth/1791-richest-customer-wealth.c
int maximumWealth(int** accounts, int n, int* c) {
    int max = 0,sum = 0;
    for(int i = 0; i < n ; i++){
       sum = 0;
       for(int j = 0; j < *c; j++){
           sum += accounts[i][j]; 
       }
       if(max < sum) max = sum;  
    }
    return max;
}

File: 1791-richest-customer-wealth/1791-richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int ans = Integer.MIN_VALUE;
        int sum = 0;
        for(int[] nums : accounts){
            sum = 0;
            for(int n : nums){
                sum += n;
            }
            if(ans < sum) ans = sum;
        }
        return ans;
    }
}